std/zip.d

```diff
@@ -343,8 +340,9 @@ public:
     @property @safe @nogc pure nothrow uint diskStartDir() const { return 0; }
 
     /// Read Only: number of ArchiveMembers in the directory.
-    @property @safe @nogc pure nothrow uint numEntries() const { return _numEntries; }
-    @property @safe @nogc pure nothrow uint totalEntries() const { return _totalEntries; }    /// ditto
+    deprecated("Use totalEntries instead; will be removed in 2.099.0")
+    @property @safe @nogc pure nothrow uint numEntries() const { return cast(uint) _directory.length; }
+    @property @safe @nogc pure nothrow uint totalEntries() const { return cast(uint) _directory.length; }    /// ditto
```

Comment:
I question the need to mark `numEntries` for removal. Every existing D program that used std.zip and wrote `numEntries` instead of `totalEntries` will need to be manually changed or else fail to compile within the next year, but there will be no improvement in the behavior of those programs in exchange for this labor. This is good for keeping programmers employed but little else.

Reply:
One or the other needs to go, otherwise we will forevermore have two getters with the same value.

Reply:
What is wrong with an alias?

Reply:
IMHO programs that use `numEntries` instead of `totalEntries` probably have a potential bug. I don't think, that this bug will ever appear, but that's only, because I think, we will never add multi disk support. With multi disk support, the two entries would differ (`numEntries` is the number of entries on this disk, while `totalEntries` is the number of entries on all disks) and it's high likely that `numEntries` is the wrong one.

Having said this, I'm not sure, what's the better option...